### PR TITLE
fix: changing conversation options are displayed for personal users

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

changing conversation options are displayed for personal users

### Solutions

if self is a team member, then show the options, otherwise create the conversation with default values

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] https://github.com/wireapp/kalium/pull/1207

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/4/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt [hunk 1]
```diff
@@ -26,14 +26,17 @@ import com.wire.kalium.logic.feature.publicuser.GetAllContactsUseCase
...
-import javax.inject.Inject
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import javax.inject.Inject
 
 @Suppress("LongParameterList", "TooManyFunctions")
 @HiltViewModel
 class NewConversationViewModel @Inject constructor(
     private val createGroupConversation: CreateGroupConversationUseCase,
+    private val isSelfATeamMemberUseCase: IsSelfATeamMemberUseCase,
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt]: I think the name without `UseCase` suffix 😉

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5154476)(5154476)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5154476)(5154476)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt [hunk 2]
```diff
@@ -131,7 +138,24 @@ class NewConversationViewModel @Inject constructor(
         return false
     }
 
-    fun createGroup(shouldCheckGuests: Boolean = true) {
+    fun createGroupWithoutOption() {
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt]: I don't see it being used anywhere, can it be removed? 🤔

**Review by saleniuk — APPROVED** [review]
Just two suggestions, but besides that it seems ok to me 👌

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/6/) **failed**.

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/7/) **failed**.

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1216/8/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5161741)(5161741)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5161741)(5161741)-dev-release.apk)

**Pull request merged**